src/components/ChannelPreview/hooks/useChannelPreviewDisplayAvatar.js

```diff
@@ -9,32 +9,38 @@ import { ChatContext } from '../../../context';
  */
 export const useChannelPreviewDisplayAvatar = (channel) => {
   const { client } = useContext(ChatContext);
-  const [displayAvatar, setDisplayAvatar] = useState({});
```

Comment:
State was initialized to empty object, which caused first render of channelpreview to display empty circles on screen.